test/test_cuda.py

```diff
@@ -4553,6 +4553,45 @@ def power2_div(size, div_factor):
             # not supported with the cudaMallocAsync backend
             self.assertEqual(pow2_div2_mem - start_mem, power2_div(nbytes_big, 2))
 
+        # test roundup_power2_divisions:1 rounds to next power of 2
```

Comment:
Not an issue introduced by this PR, but for anti-flakiness we probably want the @serialTest() decorator for this test as well